crypto/evp/pmeth_lib.c

```diff
@@ -656,8 +648,7 @@ const OSSL_PARAM *EVP_PKEY_CTX_settable_params(EVP_PKEY_CTX *ctx)
             && ctx->op.ciph.cipher->settable_ctx_params != NULL)
         return ctx->op.ciph.cipher->settable_ctx_params();
     if (EVP_PKEY_CTX_IS_GEN_OP(ctx)
-            && ctx->keymgmt != NULL
-            && ctx->keymgmt->gen_settable_params != NULL)
+            && ctx->keymgmt != NULL)
```

Comment:
Er, why?

Reply:
I knew you would ask that - the util function that it calls does this check. It is different to some of the others that just call the function directly if the value is not NULL.

Reply:
Ah!